src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,151 @@
 package core.basesyntax;
 
+import core.basesyntax.model.User;
+import core.basesyntax.service.ImpossibleRegisterUserException;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorldTest {
+    private RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+    private User user = new User();
+
+    @BeforeEach
+    void setUp() {
+        user.setId(12l);
+        user.setLogin("sanya67");
+        user.setAge(18);
+        user.setPassword("123456");
+    }
+
+    @Test
+    void loginOfUserAlreadyExists_NotOk() {
+        try {
+            Assert.assertEquals(user, registrationService.register(user));
+        } catch (ImpossibleRegisterUserException e) {
+            System.out.println("this login isn`t available");
+        }
```

Comment:
Avoid using try-catch blocks for exception testing. Use `Assertions.assertThrows` instead for a more concise and clear test.